bigframes/ml/sql.py

```diff
@@ -177,6 +177,24 @@ def create_model(
         parts.append(f"AS {source_sql}")
         return "\n".join(parts)
 
+    # Model create and alter
+    def create_llm_remote_model(
```

Comment:
Looks like we can refactor to merge the create functions. Not necessarily in this PR though.

Reply:
sg!